docs/fastdds/transport/interfaces.rst

```diff
@@ -0,0 +1,190 @@
+from an interface to an unreachable destination given a concrete network architecture.
+In this situation, a user may enable the netmask filtering feature, which would prevent this behaviour by only sending
+data from a network interface to remote locators within the same subnetwork.
+
+This configuration option can be set at participant, transport and interface levels, and its possible values are:
+
+.. _netmask_filter_values:
+
+.. list-table::
+   :header-rows: 1
+   :align: left
+
+   * - Value
+     - Description
+   * - |NetmaskFilterKind::ON-api|
+     - Enable netmask filtering.
+   * - |NetmaskFilterKind::OFF-api|
+     - Disable netmask filtering.
+   * - |NetmaskFilterKind::AUTO-api|
+     - Use container's netmask filter configuration.
+
+An |NetmaskFilterKind::AUTO-api| netmask filter configuration means that its effective value will be given by that of
+its "container", which in the case of an :ref:`allowlist <interfaces_allowlist>` entry would be the transport descriptor
+where it is included, and in the case of a transport descriptor would be the participant where it is registered.
+
+Note however that not all configurations are valid; this is, for example, a transport's netmask filter configuration
+cannot be |NetmaskFilterKind::OFF-api| if it is |NetmaskFilterKind::ON-api| for the participant where this transport
+is registered.
+Likewise, the netmask filter configuration for an allowlist entry cannot be |NetmaskFilterKind::ON-api| if it is
+|NetmaskFilterKind::OFF-api| for the transport descriptor where this allowlist is defined.
+
+Due to implementation details, it is required to set `ignore_non_matching_locators` to `true`
+(see :ref:`Matching algorithm <external_locators_algorithm>`) both in :ref:`participants <wireprotocolconfigqos>`
+and :ref:`endpoints <rtpsendpointqos>` when enabling the netmask filtering feature at participant/transport level
+without defining an allowlist.
+
+Additional considerations need to be taken into account when using netmask filtering in combination with
+:ref:`external locators <external_locators>`.
+In particular, it is not possible to enable netmask filtering in all entries of an allowlist when a set of local
+external locators (at externality greater than `0`) is defined for a :ref:`participant <wireprotocolconfigqos>` or
+:ref:`endpoint <rtpsendpointqos>`.
+The reason for this is that a matching remote external locator would then (most likely) be effectively ignored, as no
+network interface would be able to reach it according to its network mask.
+
+Netmask filtering can be enabled both via C++ API or XML configuration.
+In the case of participants, the netmask filter configuration is set in C++ as follows:
+
+.. literalinclude:: /../code/DDSCodeTester.cpp
+  :language: c++
+  :start-after: //CONF-NETMASK-FILTER
+  :end-before: //!--
+  :lines: 3-4
+  :dedent: 8
+
+See :ref:`RTPS` for an example on how to configure netmask filtering for a participant through an XML profile.
```

Comment:
It could be really helpful opening another tab here with the XML participant profile example